infrastructure/server-setup/backups.yml

```diff
@@ -87,16 +87,6 @@
   vars:
     manager_hostname: "{{ groups['docker-manager-first'][0] }}"
   tasks:
-    - name: Ensure backup user is present
```

Comment:
> The Ansible version on Github contains https://github.com/ansible/ansible/issues/75352 bug which causes the created backup user to have bad permissions and the backup script gets a permission denied (public key) error when SSH'ing in during the cronjob. Instead we will instruct the implementers to manually create a backup user in documentation.

It's better to keep the setup as automated as possible so we have maximum control and predictability over the environment. Let's instead add another step that sets the permissions correctly to mitigate the issue.